projects/hip-tests/catch/unit/virtualMemoryManagement/hipMemMap.cc

```diff
@@ -288,9 +306,6 @@ TEST_CASE("Unit_hipMemMap_PhysicalMemoryReuse_MultiDev") {
     hipDevice_t device;
     HIP_CHECK(hipDeviceGet(&device, devX));
     checkVMMSupported(device);
-    hipMemAllocationProp prop{};
```

Comment:
I'm not sure why these tests in hipMemMap.cc are re-written differently.
You had the SECTION() just for the prop.type before, but now these test cases are re-written to be helper functions with a different prop.type initialized instead. Although functionally it should work, why the style change between the different tests?